### PR TITLE
Update footer.tmpl.partial

### PR DESCRIPTION
**Pull request by vyaspranjal33**
updated year 2016 - > 2018 in footer

### Prerequisites

- [ ] I have [created an issue](https://github.com/umbraco/Umbraco-CMS/issues) for the proposed changes in this PR, the link is:
- [ ] I have added steps to test this contribution in the description below

### Description

updated year in footer

### CODE REVIEW

**Comment by vyaspranjal33** [comment]
@nul800sebastiaan please review

File: apidocs/umbracotemplate/partials/footer.tmpl.partial [hunk 1]
```diff
@@ -7,7 +7,7 @@
       <span class="pull-right">
         <a href="#top">Back to top</a>
       </span>
-      <span>Copyright © 2016 Umbraco<br>Generated by <strong>DocFX</strong></span>
+      <span>Copyright © 2018 Umbraco<br>Generated by <strong>DocFX</strong></span>
```
Comment (nul800sebastiaan) [inline apidocs/umbracotemplate/partials/footer.tmpl.partial]:
```suggestion
      <span>Copyright © 2016-present Umbraco<br>Generated by <strong>DocFX</strong></span>
```

Comment (vyaspranjal33) [reply apidocs/umbracotemplate/partials/footer.tmpl.partial]: @nul800sebastiaan  applied changes

**Comment by nul800sebastiaan** [comment]
Hi there @vyaspranjal33,

First of all: **A big #H5YR** for contributing to Umbraco during Hacktoberfest! We are very thankful for the huge amount of PRs submitted, and all the amazing work you've been doing :pray:

If you haven't heard already we have had a record breaking amount of PR's in both the CMS and the Documentation repository. You can read all about the numbers in the [Umbraco Hacktoberfest Aftermath blog post](https://umbraco.com/blog/the-2018-hacktoberfest-aftermath/).

In the spirit of Hacktoberfest we've prepared some **exclusive Umbraco swag** for all our contributors - _including you_! 

[Receive your swag!](https://goo.gl/forms/smjQwEMDM5cUnz6x1) :point_left: Please follow this link to fill out and submit the form, **before November 21st 2018**.

Following this date we'll be sending out all the swag, which also means that it might not reach your doorstep before December, so please bear with us and be patient :pray:

The only thing left to say is **thank you so much for participating in Hacktoberfest**! We really appreciate the help!

Kind regards,
The Umbraco PR Team & The Documentation Curators

**Pull request merged**